src/output.cpp

```diff
@@ -893,9 +893,9 @@ void draw_item_filter_rules( const catacurses::window &win, int starty, int heig
         starty += 1 + fold_and_print( win, starty, 1, len, c_white, _( "Example: -pipe,-chunk,-steel" ) );
     }
 
-    starty += fold_and_print( win, starty, 1, len, c_white, _( "Search [c]ategory or [m]aterial:" ) );
+    starty += fold_and_print( win, starty, 1, len, c_white, _( "Search [c]ategory,[m]aterial, or [q]uality:" ) );
```

Comment:
Maybe a space after the first comma will make it look better?

Reply:
Yep, I didn't catch that. Thanks.